### PR TITLE
Centralize Viewtype literals in a single class #314

### PR DESCRIPTION
**Pull request by tortmayr**
Implementation of DiagramHelper.ViewTypes which contains all view type literals as
constants.
Refactoring of classes which made use of the plain String literals.
Remove the now obsolete RepresentationKind class.

### CODE REVIEW

File: plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/model/spi/DiagramHelper.java [hunk 1]
```diff
@@ -125,4 +125,35 @@ Command createMessageConnector(Supplier<Message> message, //
 	 */
 	Command deleteView(EObject diagramView);
 
+	public interface ViewTypes {
```
Comment (tortmayr) [inline plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/model/spi/DiagramHelper.java]: I'm not sure if this is the optimal location for the centralization component ( I followed the apporach which was used in #308 for the modifier literals)

Comment (rschnekenbu) [reply plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/model/spi/DiagramHelper.java]: I would keep it in as a separate interface, but only for historical reasons. I do not see any pros or cons for this being a separate interface.
I do not see however the removal of RepresentationKind class in the commit, with regards to the description of PR. Did I miss something?

Comment (tortmayr) [reply plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/model/spi/DiagramHelper.java]: You are right seems like I forgot to stage the RepresentationKind remove change. Thanks for catching that.

**Comment by tortmayr** [comment]
I have rebased this PR and addressed the review comments. The RepresentationKind class is now removed for real.

**Comment by planger** [comment]
@rschnekenbu Any more comments, or are you fine with this change after the new commits by Tobias? Thanks!

**Pull request merged**